src/main/java/com/percero/amqp/PerceroAgentListener.java

```diff
@@ -36,6 +36,11 @@
  */
 @Component("perceroAgentListener")
 public class PerceroAgentListener implements MessageListener {
+	
+	private static PerceroAgentListener perceroAgentListener = null;
```

Comment:
Oh, I just noticed that you were starting to use the singleton pattern a lot (which is basically a substitute for using spring). And upon further reflection, I think it is probably a good idea to remove spring as a dependency of the `syncengine` project.